libp2p/protocols/pubsub/gossipsub/behavior.nim

```diff
@@ -19,10 +19,10 @@ declareGauge(libp2p_gossipsub_cache_window_size, "the number of messages in the
 declareGauge(libp2p_gossipsub_peers_per_topic_mesh, "gossipsub peers per topic in mesh", labels = ["topic"])
 declareGauge(libp2p_gossipsub_peers_per_topic_fanout, "gossipsub peers per topic in fanout", labels = ["topic"])
 declareGauge(libp2p_gossipsub_peers_per_topic_gossipsub, "gossipsub peers per topic in gossipsub", labels = ["topic"])
-declareGauge(libp2p_gossipsub_under_dlow_topics, "number of topics below dlow")
 declareGauge(libp2p_gossipsub_under_dout_topics, "number of topics below dout")
-declareGauge(libp2p_gossipsub_under_dhigh_above_dlow_topics, "number of topics below dhigh but above dlow")
-declareGauge(libp2p_gossipsub_no_peers_topics, "number of topics without peers available")
+declareGauge(libp2p_gossipsub_no_peers_topics, "number of topics in mesh with no peers")
+declareGauge(libp2p_gossipsub_low_peers_topics, "number of topics in mesh with at least one but below dlow peers")
+declareGauge(libp2p_gossipsub_healthy_peers_topics, "number of topics in mesh with at least dlow peers (but below dhigh)")
 declareCounter(libp2p_gossipsub_above_dhigh_condition, "number of above dhigh pruning branches ran", labels = ["topic"])
```

Comment:
no longer used?

Reply:
Still used, but I forgot to move it down, nice catch again...

Reply:
I'm going to keep it separate tho from the others.. I draw it in a heatmap on its own generally

Reply:
or shall we remove it? it's quite internal/not so useful

Reply:
I think it's better replaced with a counter that counts prunes and grafts in general, during rebalancing

Reply:
merging for now so we can get a build done with correct metrics